jenkins/ring-install.sh

```diff
@@ -392,10 +416,10 @@ EOF
     fi
 
     # The following command must be run only once. It touches data on the ring, it does nothing at the connector's side
-    sudo sfused -X -c /etc/sfused.conf
-    sudo /etc/init.d/scality-sfused restart
+    sudo $(which sfused) -X -c /etc/sfused.conf
```

Comment:
Well, I don't know why, but on Centos7 the 'sfused' bin in not in PATH if you use sudo.